README.md

```diff
@@ -1,15 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Microverse project day 2
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML and CSS
+- Frameworks: N/A
 
 ## Live Demo (if available)
```

Comment:
- [ ] It would be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. This includes git clone URL, cd project_name, etc.

- [ ] This is a solo project and not a pair programming. Kindly remove the  **Author2** from the ## Authors section